apps/explorer/src/app/routes/parties/home/index.tsx

```diff
@@ -28,21 +29,21 @@ export const JumpToParty = () => {
     <form onSubmit={handleSubmit}>
       <label
         htmlFor="block-input"
-        className="block uppercase text-h5 font-bold"
+        className="block uppercase text-h5 font-bold mb-4"
       >
         Go to party
       </label>
-      <input
-        id="block-input"
-        name="partyId"
-        placeholder="Party id"
-        className="bg-white-25 border-white border px-8 py-4 placeholder-white-60"
-      />
-      <input
-        className="border-white border px-28 py-4 cursor-pointer"
-        type="submit"
-        value="Go"
-      />
+      <div className="flex">
+        <Input
+          id="block-input"
+          name="partyId"
+          placeholder="Party id"
+          className="max-w-[200px]"
+        />
+        <Button variant="secondary" type="submit">
+          Go
+        </Button>
+      </div>
     </form>
```

Comment:
minor (non-blocking): can we make this generic that takes a label, place holder and submit handler and then consume it from jump to block. Possibly could also use for search 🤔

Reply:
Nice idea!